openprescribing/measure_definitions/toothpaste

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "Toothpaste Prescribing",
+  "title": [
+    "Prescribing of toothpaste items cost per 1000 patients"
+  ],
+  "description": [
+    "Prescribing of toothpaste items cost per 1000 patient"
+  ],
+  "why_it_matters": [
+    "OpenPrescribing currently has data on GP prescriptions and does not have information on dentists prescribing. ",
+    "Dentists are best placed to assess, prescribe and monitor dental problems and it is recommended that they should ",
+    "issue any prescriptions such as tootpastes for dental conditions. Additionally ",
+    "NHS England [recommends](https://www.england.nhs.uk/wp-content/uploads/2018/03/otc-guidance-for-ccgs.pdf) that there should ",
```

Comment:
The `[text](url)` syntax is markdown, not HTML.  You'll need to change this to `<a href="https://www.england.nhs.uk/wp-content/uploads/2018/03/otc-guidance-for-ccgs.pdf\">recommends</a>`.